test/common/upstream/load_balancer_impl_test.cc

```diff
@@ -33,6 +34,34 @@ namespace Envoy {
 namespace Upstream {
 namespace {
 
+static constexpr uint32_t UnhealthyStatus = 1u << static_cast<size_t>(Host::Health::Unhealthy);
+static constexpr uint32_t DegradedStatus = 1u << static_cast<size_t>(Host::Health::Degraded);
+static constexpr uint32_t HealthyStatus = 1u << static_cast<size_t>(Host::Health::Healthy);
+
+LoadBalancerContext::OverrideHostStatus createOverrideHostStatus(
```

Comment:
Maybe I'm missing something but this doesn't seem to be used?

Reply:
I will check it. Thanks.

Reply:
It is used by some new added test case.

https://github.com/wbpcode/envoy/blob/0420d3012a97ee97948bfe824b3f2015725941ca/test/common/upstream/load_balancer_impl_test.cc#L2051-L2056

Reply:
Those tests seem to be testing `createOverrideHostStatus` itself, can we just remove it nothing is actually using it?

Reply:
Ha, you are right. :rofl: